src/app/containers/ArticleMain/index.jsx

```diff
@@ -22,17 +26,50 @@ const componentsToRender = {
 };
 
 const ArticleMain = ({ articleData }) => {
-  const { content, metadata, promo } = articleData;
-  const { blocks } = content.model;
+  const {
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+    articleAuthor,
+  } = useContext(ServiceContext);
+
+  const processedData = processArticleData({
+    ...articleData,
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+  });
 
   return (
     <>
       <ATIAnalytics data={articleData} />
+
       <ChartbeatAnalytics data={articleData} />
-      <MetadataContainer metadata={metadata} promo={promo} />
+
+      <MetadataContainer
+        metadata={path(['metadata'], articleData)}
+        promo={path(['promo'], articleData)}
+        pageSpecificLinkedData={path(
```

Comment:
Rather than passing the data, can you pass the function that makes the page specific data, and MetadataContainer calls it? This files getting very cluttered

Reply:
I would then also need to pass article specific arguments to go into the function down.